notebook/static/notebook/js/textcell.js

```diff
@@ -227,7 +227,8 @@ define([
          *          notebook: Notebook instance
          */
         options = options || {};
-        var config = utils.mergeopt(MarkdownCell, {});
+        var config = new configmod.ConfigSection('markdown',
+                utils.mergeopt(MarkdownCell, options.config, {}));
```

Comment:
The second argument to ConfigSection is just for the `{base_url: '...'}` parameter. It doesn't take any defaults (that's what `ConfigWithDefaults` does).

I didn't get round to properly converting the cell options to the new config system, because it looked like it was going to be fiddly. As @Carreau mentioned, we also need to decide whether we still need config that the different cell types can inherit and override;  it makes things easier if we don't.